vulnerabilities/migrations/0077_vulnerability_exploitability_and_more.py

```diff
@@ -0,0 +1,33 @@
+# Generated by Django 4.2.16 on 2024-11-08 14:07
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("vulnerabilities", "0076_alter_packagechangelog_software_version_and_more"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="vulnerability",
+            name="exploitability",
+            field=models.DecimalField(
+                decimal_places=2,
+                help_text="Exploitability refers to the potential or probability of a software package vulnerability being \n        exploited by malicious actors to compromise systems, applications, or networks. \n        It is determined automatically by the discovery of exploits.",
+                max_digits=4,
+                null=True,
+            ),
+        ),
+        migrations.AddField(
+            model_name="vulnerability",
+            name="weighted_severity",
+            field=models.DecimalField(
+                decimal_places=2,
+                help_text="Weighted Severity is the maximum value obtained when each Severity is multiplied by its associated Weight/10.",
+                max_digits=4,
```

Comment:
"Weighted severity is the highest value calculated by multiplying each severity by its corresponding weight, divided by 10."